### PR TITLE
Online DQM  update of Pixel  folder

### PR DESCRIPTION
**Pull request by Payal7890**
We have removed few plots of Online DQM GUI from PixelPhase1 workspace (Quick Collection) and Shift workspaces (PixelPhase1 folder) in agreement with Pixel Operation team, in order to simplify the work of the Online DQM shifter. 

We tested the changes in the private GUI prova1234, using runs 380066 and 380074.

To connect to the private GUI outside of CERN network: tunneling (and proxy in browser, with same port number) are needed:

ssh -D <port number>  [CERNUSERNAME@lxplus.cern.ch](mailto:CERNUSERNAME@lxplus.cern.ch) 

We hereby attach the tinyURL links before (Official, GUI) and after the changes (New, prova1234 GUI) for the run 380066.

1) PixelPhase 1 (Quick Collection):
Original: https://tinyurl.com/2jh6wy8e
New:  :   https://tinyurl.com/4zvb5mx9

2)  Shift (PixelPhase1):
Original: https://tinyurl.com/dydxe649
New: https://tinyurl.com/3mcdvuu2

For both Online and Offline Pixel Summary maps, the bin size of the z-axis was changed in order to have the green color in the range from 0.9 to 1.0.

### CODE REVIEW

File: dqmgui/workspaces-online.py [hunk 1]
```diff
@@ -148,34 +148,34 @@
...
                   #'PixelPhase1/Layouts/01a - PixelPhase1_Event_Rate',
                   'PixelPhase1/Layouts/00a - PixelPhase1 FED Occupancy vs Lumi Sections',
-                  'PixelPhase1/Layouts/00b - PixelPhase1_Error_Summary',
-                  'PixelPhase1/Layouts/01c - PixelPhase1 Cluster Size vs Cluster Eta',
-                  'PixelPhase1/Layouts/02 - PixelPhase1_Digi_ADC_Barrel',
-                  'PixelPhase1/Layouts/03 - PixelPhase1_Digi_ADC_Endcap',
-                  'PixelPhase1/Layouts/04 - PixelPhase1_Cluster_Number',
-                  'PixelPhase1/Layouts/05 - PixelPhase1_Cluster_Charge',
-                  'PixelPhase1/Layouts/06 - PixelPhase1_Cluster_Size',
+                  #'PixelPhase1/Layouts/00b - PixelPhase1_Error_Summary',
```
Comment (nothingface0) [inline dqmgui/workspaces-online.py:151]: @Payal7890 Did you mean to remove those workspaces from the PixelPhase1 layouts? This doesn't look like it has any effect, you will probably need to modify the `pixelPhase1-layouts.py` file instead.

Comment (nothingface0) [reply dqmgui/workspaces-online.py:151]: From what I can see, you will also need to modify the `pixelPhase1-layouts.py` as well to completely remove them.

Comment (nothingface0) [reply dqmgui/workspaces-online.py:151]: @Payal7890 kind reminder for this.

**Comment by nothingface0** [comment]
@Payal7890 this will require some more modifications from your side, as another, already merged PR, has created conflicts.

**Comment by Payal7890** [comment]
Hello,

   I will fix it as soon as possible and get back to you. Thanks for your
understanding.


Best
Payal

**Comment by Payal7890** [comment]
Hello @nothingface0, we only intend to delete those plots you mentioned above from the "Quick Collection" folder. We wanted to keep 000,00a,19, 20, 24, 27, 28, 30,36, 37,43a,b as you can see  here: https://tinyurl.com/4zvb5mx9. We don't want to delete anything from the PixelPhase1 workspace completely. Please let me know if that makes sense.

**Comment by alaperto** [comment]
Hi @nothingface0 , can you update the playback machine? We just updated the code (to reintroduce some plots). 

Thanks

**Comment by nothingface0** [comment]
@alaperto It's updated, feel free to take a look

**Comment by alaperto** [comment]
Ok, looks good!

**Comment by alaperto** [comment]
Hi @nothingface0 , when this will be deployed?

**Comment by alaperto** [comment]
Reminder for @nothingface0 
Waiting for deployment

**Comment by nothingface0** [comment]
@alaperto Sorry for the delay, I was on vacation. We will update the online DQMGUI this week, along with a CMSSW release which has been planned. Is this also needed for Offline?

**Comment by nothingface0** [comment]
@alaperto This is now deployed in the Online DQMGUI

**Pull request merged**